rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, person, book)
+    @date = date
+    @book = book
+    @person = person
+    person.rentals << self
+    book.rentals << self
+  end
+end
```

Comment:
- [ ] Great job implementing the required changes for this milestone, however when listing rentals for any given person, the app lists duplicate entries which display `No rentals found for that ID`, kindly fix this bug
![image](https://user-images.githubusercontent.com/81932478/224491386-adedf57a-c052-4040-8c02-17d7defaeee5.png)